internal/contour/cluster.go

```diff
@@ -17,8 +17,9 @@ import (
 	"sort"
 	"sync"
 
+	rsrc "github.com/envoyproxy/go-control-plane/pkg/resource/v2"
+
```

Comment:
Do we have a standard style for grouping imports?

Reply:
I don't know if we have a standard, I copied this from the go-control-plane code.

Reply:
I renamed it to `resource`. I didn't think the other alias made sense.